kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/WatchConnectionManager.java

```diff
@@ -107,7 +85,7 @@ public WatchConnectionManager(final OkHttpClient client, final BaseOperation<T,
     this(client, baseOperation, listOptions, watcher, reconnectInterval, reconnectLimit, websocketTimeout, 5);
   }
 
-  private final void runWatch() {
+  private void runWatch() {
     logger.debug("Connecting websocket ... {}", this);
```

Comment:
Output target API URL instead of rather useless manager object reference
```suggestion
    logger.debug("Connecting websocket to {}...", requestURL);
```